pkg/hub/submarineragent/controller_test.go

```diff
@@ -334,6 +334,9 @@ func newTestDriver() *testDriver {
 		providerFactory := cloudFake.NewMockProviderFactory(t.mockCtrl)
 		providerFactory.EXPECT().Get(gomock.Any(), gomock.Any(), gomock.Any()).Return(t.cloudProvider, nil).AnyTimes()
 
+		// TODO: Investigate why when tests are a bit slower (e.g run with -v flag) fail due to this being expected.
```

Comment:
I'm investigating this so we don't need this comment.

Reply:
I'd rather merge before 2.6.0 code freeze, I don't mind removing the workaround but UT might fail

Reply:
The test failure is fixed by #453 (see https://github.com/stolostron/submariner-addon/issues/455#issuecomment-1195605284).  I'd rather not leave a stale comment.